simulationTool/locales/en/additional.json

```diff
@@ -15,26 +17,45 @@
         "chartType": "Chart type",
         "charts": "Charts",
         "completed": "Completed",
+        "contact": "Contact",
+        "contactInfo": "Do you still have questions? Please write an e-mail to ",
         "confirm": "Confirm",
         "created": "Created",
         "createEnsemble": "Create ensemble",
+        "createEnsembleContent": "After you have clicked on \"Add ensemble\" in the ensemble overview, you can create a new ensemble here. Name it as you wish and click on \"Create empty ensemble\" so that it appears in your ensemble overview and is available for further editing.",
+        "createEnsembleContent2": "As soon as you click on the plus symbol, you can select the models that your ensemble should contain. Your model selection is displayed in an overview from which you can easily remove models. You can find further settings under the \"Configure\" function. Click on “Simulate ensemble” to calculate the displayed number of scenarios from the models. These scenarios are displayed in the scenario overview.",
+        "createEnsembleContent3": "The model-dependent configurations consist on the one hand of the display of the model uncertainties and on the other hand of the settings of the number of scenarios to be calculated and the selection method of the samples from the uncertainty ranges.",
+        "createEnsembleHint": "Models contain unavoidable mathematical uncertainties that comprise a numerical range instead of a clear numerical value. When an ensemble is simulated, various input parameters from these uncertainty ranges are automatically used to calculate the scenarios.",
+        "createScenario": "Create scenario",
+        "createScenarioContent": "The heart of the Scenario Explorer lies in the execution of simulations. On this page, you can create your own scenario by naming it individually and setting the required input parameters using a wide range of input options. Click on \"Run scenario\" to start the calculations and you will be automatically redirected to the scenario detail page.",
```

Comment:
```suggestion
        "createScenarioContent": "The heart of the Scenario Explorer lies in the execution of simulations. On this page, you can create your own scenario by naming it individually and setting the required input parameters. Click on \"Run scenario\" to start the calculations and you will be automatically redirected to the scenario detail page.",
```

Reply:
The German translations are derived from the mockups. I can adjust my English translations but would prefer to avoid changing the German translations.